org.libvips.vipsdisp.json

```
@@ -0,0 +1,349 @@
+{
+    "app-id" : "org.libvips.vipsdisp",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "vipsdisp",
```

```suggestion
    "command" : "vipsdisp",
    "rename-desktop-file": "vipsdisp.desktop",
```

The desktop file isn't properly named. So this will take care of it.

---

Ahhhh, brilliant, thank you. I'll try that.